lib/crypto/eosdart/src/models/account.g.dart

```diff
@@ -118,16 +82,13 @@ Map<String, dynamic> _$PermissionToJson(Permission instance) =>
 RequiredAuth _$RequiredAuthFromJson(Map<String, dynamic> json) {
   return RequiredAuth()
     ..threshold = json['threshold'] as int?
-    ..keys = (json['keys'] as List?)
-        ?.map((e) =>
-            e == null ? null : AuthKey.fromJson(e as Map<String, dynamic>))
-        .toList()
-    ..accounts = (json['accounts'] as List?) as List<Object>?
-    ..waits = (json['waits'] as List?) as List<Object>?;
+    ..keys =
+        (json['keys'] as List?)?.map((e) => e == null ? null : AuthKey.fromJson(e as Map<String, dynamic>)).toList()
+    ..accounts = List<Object?>.from(json['accounts'])
```

Comment:
will this work if json['accounts'] is null?

I don't think this can ever be null, but would feel better if that case continued working - it worked in the original code...